src/pages/sponsors/sponsor-cart-tab/components/edit-form/index.js

```
@@ -0,0 +1,353 @@
+      break;
+    }
+  }
+
+  return schema;
+};
+
+const buildInitialValues = (form, timeZone) => {
+  const items = form?.items || [];
+
+  const initialValues = items.reduce((acc, item) => {
+    item.meta_fields.map((f) => {
```

Biome (2.4.4)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)